src/validate/request.js

```diff
@@ -135,14 +144,7 @@ function relation(node, i) {
     node.postData &&
     !emptyObject(node.postData)
   ) {
-    throw new InvalidArchiveError(
-      createErrorParams({
-        name: 'InvalidRequestData',
-        index: i,
-        path: 'postData',
-      }),
-      `Request postData usage with GET requests is prohibited`
-    )
+    console.warn(`[WARN] GET request has postData object (${i})`)
```

Comment:
I think it's pretty safe to simply ignore the postData when it's a GET-request. It is obviously an error in the thing that generated the HAR file.